native/cocos/audio/include/AudioMacros.h

```diff
@@ -75,3 +75,7 @@
         CC_LOG_DEBUG("(" QUOTEME(condition) ") failed, message: " fmt, ##__VA_ARGS__); \
         break;                                                                         \
     }
+#ifndef __BEGIN_DECLS
```

Comment:
`__BEGIN_DECLS` is a built-in macro, we should define our own version.

Reply:
1